docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    area = dy0.data*np.tile(dx0, (len(ds['longitude'].data),1))
</pre><p>I would change <code>dy0.data</code>  to <code>dy0</code>  since the latter is of type <code>np.float64</code>  (and not an xarray DataArray).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>